packages/field-base/test/input-control-mixin.test.js

```diff
@@ -80,6 +83,30 @@ const runTests = (defineHelper, baseMixin) => {
       expect(spy.calledOnce).to.be.true;
     });
 
+    it('should clear the input value on clear button mousedown', async () => {
```

Comment:
These tests should be probably placed in `clear-button-mixin.test.js`.

Reply:
Done.